components/automate-elasticsearch/habitat/hooks/post-run

```diff
@@ -0,0 +1,41 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash
+
```

Comment:
do we need to set -x

Reply:
Do you mean `set -x`?

Reply:
yes, that, the error one

Reply:
I'm intentionally not failing at all in the post-run hook because it's a best effort attempt. Since there is no backoff of retries and hab doesn't currently reap hooks when a service is unloaded, that seems like the safest approach.